include/h2o.h

```diff
@@ -1028,8 +1028,18 @@ struct st_h2o_req_t {
         struct timeval request_body_begin_at;
         struct timeval response_start_at;
         struct timeval response_end_at;
-        h2o_httpclient_timings_t proxy;
     } timestamps;
+    /**
+     * proxy stats
+     */
+    struct {
+        struct {
+            size_t total;
```

Comment:
Two things:
* number of bytes being sent should be represented by `uint64_t` rather than `size_t` (which could be 32-bit)
* do we need `total`? Can't that be inferred from `header` and `body`?

Reply:
I think we need `total` because
1. for proxy protocol
2. for protocols (ex: http2) which has other data transmitted than header and body